logging/logger_manager.go

```diff
@@ -41,3 +41,20 @@ func (lr *loggerRegistry) updateLoggerLevel(name string, level Level) error {
 	logger.SetLevel(level)
 	return nil
 }
+
+// Exported Functions specifically for use on global logger manager.
```

Comment:
Could we add a `TODO` and a ticket number above the declaration of `loggerManager` to make the `loggerManager` not a global and instead a variable on `localRobot`? Or, make it a nonglobal as part of this PR if it's a somewhat trivial change?

Reply:
the `loggerManager` is already non-global, we just have functions that operate specifically on that variable. I'll make a ticket about making it a variable on `localRobot`

Reply:
done!